public/index.html

```diff
@@ -15,6 +15,35 @@
     <title>Survey Taker Application</title>
   </head>
   <body>
+        <!-- Navigation Bar -->
+        <div class="nav-container">
+          <nav class="navbar navigation-bar" role="navigation" aria-label="main navigation">
+              <div class="navbar-brand">
+                <a class="navbar-item" href="#">
+                  <img src="assets/images/girlscript_logo.png"  alt="GirlScript Banglore Logo">
+                  <h1 class="nav-title navbar-item"></h1>
+                </a>
+                <div class="navbar-burger burger" data-target="navbarExampleTransparentExample">
+                  <span></span>
+                  <span></span>
+                  <span></span>
+                </div>
+              </div>
+
+              <div id="navbarExampleTransparentExample" class="navbar-menu">
+                <div class="navbar-end">
+
+                  <a class="item-nav navbar-item" href="#">
+                    About Us
+                  </a>
+                  <a class="item-nav navbar-item" href="#">
+                    Contact Us
+                  </a>
+                </div>
+              </div>
+          </nav>
+        </div>
+        <!-- Navigation Bar End -->
```

Comment:
These changes should be inside the `common/navbar/navbar.js` file.

Reply:
ok sir. Will update and work on these.